app/src/main/java/com/nextcloud/talk/utils/bundle/BundleKeys.kt

```diff
@@ -80,4 +80,6 @@ object BundleKeys {
     const val KEY_CREDENTIALS: String = "KEY_CREDENTIALS"
     const val KEY_FIELD_MAP: String = "KEY_FIELD_MAP"
     const val KEY_CHAT_URL: String = "KEY_CHAT_URL"
+    const val KEY_IS_ROOM_ONE_TO_ONE: String = "KEY_IS_ROOM_ONE_TO_ONE"
```

Comment:
there is already a bundle key with 
KEY_ROOM_ONE_TO_ONE

just check if it's set correctly in ChatActivity or CallNotificationActivity set so you can be sure it's in place when you get it in CallActivity.

Reply:
The KEY_ROOM_ONE_TO_ONE must also be set in CallNotificationActivity, maybe also be passed through it..
Otherwise the clickListeners may not be triggered when the bundle keys are not set whenever a call is received via notification.